github/ipfs.yml

```diff
@@ -6524,8 +6524,8 @@ teams:
     members:
       member:
         - aschmahmann
-        - BigLep
         - Jorropo
+        - lidel
```

Comment:
was this intentional?

Reply:
Yeah, this was intentional as figuring Lidel should have access here, but it is an unrelated change.  I'll remove it for now.  It can always be added later.